app/controllers/unconfirmed_users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UnconfirmedUsersController < ApplicationController
+  def delete
+    count = CleanUsersService.call
```

Comment:
Love the new service! <3  How about a more descriptive name like `DeleteUnconfirmedUsers`?

Reply:
Thx for the quick response - so you mean change the controller name?  If so, is `DeleteUnconfirmedUsersController` ok?

Reply:
Ohhhhh.....I think you mean the service name!  On it...